react/week1/todolist/src/Todolist.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+const TodoListItems = (props) => {
+  const {description, deadline} = props.todo
+  return (
+    <li>
+      <h1>
+        {description} : {deadline}
+      </h1>
+    </li>
+  );
+};
```

Comment:
Good that you separated your TodoList from each item :+1:  It'd be even better if you created a new file for your Item component. You use only two components in this case but it's better to get used to keep them separated for a more clear structure!
Also, consider using singular for the item. It's one item component and you're just creating many from you data.

Reply:
I created a separate file for TodoItem as suggested.